### DISCUSSION TITLE
How to load a page with vercel

### DISCUSSION THREAD

**Discussion by RealYusufIsmail** [vuepress/core#1117] [Q&A] [answered]
Currently, i have 
<img width="909" alt="Screenshot 2022-10-02 at 14 48 25" src="https://user-images.githubusercontent.com/67903886/193457426-08bf4d3f-42f3-4c1b-874c-a23d173fc8d3.png">

When I open my site I see 
<img width="981" alt="Screenshot 2022-10-02 at 14 48 46" src="https://user-images.githubusercontent.com/67903886/193457447-220c7ade-3889-47ea-92b4-f7dcc4d1b098.png">
https://www.ydwk.org

**Comment by Mister-Hope** [comment, marked as answer]
Build command `vuepress build docs`, output folder default is `docs/.vuepress/dist`.

Try not to be too noob. We already provide similar settings with other service.

**Reply by RealYusufIsmail** [reply]
lol ok thx

**Reply by Mister-Hope** [reply]
Just realize we already have a description in docs: https://v2.vuepress.vuejs.org/guide/deployment.html#vercel

**Reply by RealYusufIsmail** [reply]
that was not there when i last checked

**Comment by Mister-Hope** [comment]
The page last update time was on Apirl, so the question takes you half a year?

**Reply by RealYusufIsmail** [reply]
no i checked yesterday. Must have looked at the wrong thing.
